source/transfer_list.rst

```diff
@@ -246,13 +246,18 @@ reservations of tag ranges for specific use.
 The {0xff_f000, ..., 0xff_ffff} range is reserved for non-standardized use.
 Anyone is free to use tags from that range for any custom TE layout without
 adding their definitions to this specification first. The use of this range is
-*strongly discouraged* for anything other than local experiments or code that
-will only ever be used in closed-source components owned by the entity
-controlling the entire final firmware image. In particular, a creator of a TE
-that just contains platform-specific data or internal structures specific to
-a single firmware implementation, should allocate a standardized tag for it
-in this specification -- using the non-standardized range is strongly
-discouraged. Since standards often emerge organically, the goal is to
+*strongly discouraged* for anything other than:
+
+* local experiments,
+
+* code that will only ever be used in closed-source components owned by the
+  entity controlling the entire final firmware image, or
```

Comment:
IMO that defeats the purpose of this specification, which is to allow interoperable components.

There are only two cases, really:
1. The undocumented information is necessary for booting, iwc the device is a brick without those secret pieces. There is no point in putting it in this spec and it should certainly not be allocated a tag
2. The undocumented information is not necessary for booting, iwc we should leave it out.

Reply:
> HI @sjg20. As stated [here](https://github.com/FirmwareHandoff/firmware_handoff/pull/57#issuecomment-2657193037), the latest proposal is to withdraw this PR and resurrect #37 with improved TF-A documentation of the bindings (including a stable link). Julius said [here](https://github.com/jwerner-chromium) he was OK with this approach. Are you? It's really not an option for us to turn this into a giant C struct and document here.

If it is a giant C struct then yes it doesn't belong in Firmware Handoff, which it for simple things. It should use the devicetree.